src/client.rs

```diff
@@ -97,6 +97,11 @@ command! { impl IrcClientIncomingCommand / IrcClientIncomingCommandError
         | ERR_ERRONEUSNICKNAME |
         | RPL_YOURESERVICE     | RPL_YOURHOST
         | RPL_MYINFO
+
+    /// > La session d'un client se termine par unn message `QUIT`. Le
+    /// > serveur en accuse la réception en envoyant un message `ERROR` au
+    /// > client.
+    <- QUIT { }
```

Comment:
On peut peut-être omettre ces accolades { } pour ce genre de cas ; où une commande peut ne prendre aucun argument, comme suit:

```suggestion
    <- QUIT
```

Reply:
Yay !